app/models/legal_aid_application.rb

```diff
@@ -212,6 +212,10 @@ def default_delegated_functions_cost_limitation
     lead_proceeding_type.default_cost_limitation_delegated_functions
   end
 
+  def passported?
+    benefit_check_result&.positive? || false
+  end
```

Comment:
There's already `applicant_receives_benefit?` on line 112

Reply:
Have gone with this name.  It's not quite accurate, because the applicant could receive a benefit which still does not qualify them for a passported legal aid application, but will do for now.  Have repurposed `passported?` test to test the `applicant_receives_benefit?` method, and changed it to return false rather than nil if there is no benefit check record.  But prehaps raising an exeption is the right thing to do?